internal/dataplane/parser/ingressrules.go

```diff
@@ -159,11 +159,15 @@ func getK8sServicesForBackends(
 	// retreieve that backend and capture any Kong annotations its using.
 	k8sServices := make([]*corev1.Service, 0, len(backends))
 	for _, backend := range backends {
-		k8sService, err := storer.GetService(namespace, backend.Name)
+		backendNamespace := namespace
+		if len(backend.Namespace) > 0 {
```

Comment:
nit: it's not really a problem because it's idiomatic and the compiler actually will produce the same thing in either case, but was there some specific reason to use a len check here instead of `== ""`? From my perspective in this case `== ""` is more clear at a glance, but again it doesn't matter all that much I only noticed it because we don't generally use this form.

Reply:
Not really any reason, no.